core/controllers/reader.py

```diff
@@ -346,7 +410,12 @@ class ReaderFeedbackHandler(base.BaseHandler):
 
     @require_playable
     def post(self, exploration_id):
-        """Handles POST requests."""
+        """Handles POST requests.
+        Records a thread_id of the feedback from a learner.
+        Args:
+            exploration_id: str. The id for the exploration the learner wants
```

Comment:
As above, this can be "The ID of the exploration." once the general purpose is clarified in the main docstring.